server/boards/tests/routes/pin-board.test.ts

```diff
@@ -36,15 +38,15 @@ describe("Tests pin boards REST API", () => {
     expect(res.status).toBe(404);
   });
 
-  //   test("Should return 403 if user does not have required permissions", async () => {
```

Comment:
Commented out lines?

Reply:
Those were your commented out lines, the green is my changes. You had pre-written and commented out permissions-related tests in a few places, I assumed knowing that there would be relevant permissions eventually. I updated them and put them to use.

Reply:
Ooops my bad :) that's why reviewing on mobile should be done sparingly!